The name of the serial device has changed since FreeBSD 6.0
http://lists.freebsd.org/pipermail/freebsd-questions/2006-September/132027.html
http://www.freebsd.org/doc/en/books/faq/serial.html
Index: avrdude/configure.ac
===================================================================
--- avrdude.orig/configure.ac	2010-01-17 13:14:31.330413278 +0100
+++ avrdude/configure.ac	2010-01-17 13:15:03.213498373 +0100
@@ -198,11 +198,11 @@
 		;;
 	i[[3456]]86-*-*freebsd*|amd64-*-*freebsd*)
 		DEFAULT_PAR_PORT="/dev/ppi0"
-		DEFAULT_SER_PORT="/dev/cuaa0"
+		DEFAULT_SER_PORT="/dev/cuad0"
 		;;
 	*-*-*freebsd*)
 		DEFAULT_PAR_PORT="unknown"
-		DEFAULT_SER_PORT="/dev/cuaa0"
+		DEFAULT_SER_PORT="/dev/cuad0"
 		;;
 	*-*-solaris*)
 		DEFAULT_PAR_PORT="/dev/printers/0"
Index: avrdude/doc/avrdude.info
===================================================================
--- avrdude.orig/doc/avrdude.info	2010-01-17 13:14:31.385497113 +0100
+++ avrdude/doc/avrdude.info	2010-01-17 13:15:12.293492927 +0100
@@ -1714,7 +1714,7 @@
 
 *System*               *Default Parallel      *Default Serial Port*
                        Port*                  
-FreeBSD                `/dev/ppi0'            `/dev/cuaa0'
+FreeBSD                `/dev/ppi0'            `/dev/cuad0'
 Linux                  `/dev/parport0'        `/dev/ttyS0'
 Solaris                `/dev/printers/0'      `/dev/term/a'
 
Index: avrdude/doc/avrdude.texi
===================================================================
--- avrdude.orig/doc/avrdude.texi	2010-01-17 13:14:31.425497182 +0100
+++ avrdude/doc/avrdude.texi	2010-01-17 13:15:18.193502809 +0100
@@ -1904,7 +1904,7 @@
   @tab @strong{Default Serial Port}
 @item FreeBSD
   @tab @code{/dev/ppi0}
-  @tab @code{/dev/cuaa0}
+  @tab @code{/dev/cuad0}
 @item Linux
   @tab @code{/dev/parport0}
   @tab @code{/dev/ttyS0}
